KsApi/extensions/ApolloClient+RAC.swift

```diff
@@ -62,4 +100,42 @@ extension ApolloClient: ApolloClientType {
       }
     }
   }
+
+  /**
+   Performs a GraphQL mutation request with a given mutation.
+
+   - parameter mutation: The `Mutation` to perform.
+
+   - returns: A `AnyPublisher` generic over `Mutation.Data` and `ErrorEnvelope`.
+   */
+  public func perform<Mutation: GraphQLMutation>(
+    mutation: Mutation
+  ) -> AnyPublisher<Mutation.Data, ErrorEnvelope> {
+    let fetchSubject: PassthroughSubject<Mutation.Data, ErrorEnvelope> = .init()
```

Comment:
One interesting note - `PassthroughSubject` just drops events if nobody is subscribed, and this fires off the networking request immediately. I don't think that will be a problem for these network requests, since presumably they'll be created + subscribed at the same time. But could be an edge case to watch out for.